lang/java/compiler/src/main/velocity/org/apache/avro/compiler/specific/templates/java/classic/record.vm

```diff
@@ -310,7 +310,7 @@ static {
 
     /** Creates a new Builder */
     private Builder() {
-      super(SCHEMA$);
+      super(SCHEMA$,MODEL$);
```

Comment:
```suggestion
      super(SCHEMA$, MODEL$);
```
This is really nit-picky and I apologise... I don't think it's really important that we generate beautified code, but there's no reason to avoid the code formatting rules from the project either!

We should have a space between parameters in a method or constructor call.

Reply:
Not nit-picky at all, but just shows that you're doing code reviews the way they're supposed to be done. I entirely missed that one (or rather, spotless:apply didn't catch it. Will fix that :)